scripts/new_bare_repo

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+usage()
+{
+cat << EOF
+usage: new_bare_repo <hutch> <repo name>
+
+Script for automating the tedious process of integrating an existing IOC into our git bare repo scheme
+Run in the top level of your IOC
+
+Ex. new_bare_repo rix new_device
+(Creates a new bare repo in /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/rix/ named "new_device.git")
+
+You also might need to run kinit and aklog before hand!
+
+EOF
+}
+
+if [ $# -lt 1 ]; then
```

Comment:
Should this be a 2? Or does this count differently than I think it does? I'm expect as-written that the script would let you run with a blank repo which might behave oddly.

I think the help check needs to become the first part of this if/else and this needs to be boosted to 2

Reply:
No, you are reading that correctly, that should be 2